docs/posts/ibis-version-9.0.0-release/index.qmd

```diff
@@ -0,0 +1,351 @@
+:::
+
+```{python}
+from ibis.expr.visualize import to_graph
+
+expr = (
+    t.group_by("species", "island")
+    .agg(count=t.count())
+    .order_by(ibis._["count"].desc())
+)
+
+to_graph(
+    expr,
+    label_edges=True,
+    node_attr={"shape": "hexagon", "color": "green", "fontname": "Roboto Mono"},
+    edge_attr={"fontsize": "12", "fontname": "Comic Sans MS"},
+)
+```
+
+### Polars input and output
+
+You can now directly output a Polars dataframe from an Ibis table (instead of
+going through Apache Arrow as previously possible):
+
+```{python}
+t.to_polars()
+```
+
+And you can directly input a Polars dataframe into an Ibis memtable:
+
+```{python}
+ibis.memtable(t.to_polars())
+```
+
+These conversions are efficient and improve usability of Ibis and Polars
+together.
+
+### Scalar subqueries from expressions
+
+You can now create scalar subqueries from expressions:
+
+```{python}
+t.select("species").limit(1).as_scalar()
+```
+
+### Disconnect
+
+You can now explicitly disconnect from a backend. First, connect to a backend:
+
+```{python}
+con = ibis.connect("duckdb://")
+con.list_tables()
+```
+
+Then you can disconnect. Trying to use the connection after will raise an error:
+
+```{python}
+# | error: true
+con.disconnect()
+con.list_tables()
+```
+
+### Today
+
+You can now get the current date more easily:
+
+```{python}
+ibis.today()
+```
+
+And use it in expressions:
+
+```{python}
+t.mutate(ingested_at=ibis.today()).relocate("ingested_at")
+```
+
+### UUIDs
+
+You can now generate universally unique identifiers (UUIDs) in Ibis more easily:
+
+```{python}
+ibis.uuid()
+```
+
+And use them in expressions:
+
+```{python}
+t.mutate(id=ibis.uuid()).relocate("id")
+```
+
+### TopK includes NULLs
+
+This could technically be considered a breaking change or bug fix. Regardless,
+`topk` now includes NULLs by default:
+
+```{python}
+t["sex"].topk(3)
+```
+
+### Order and group by variadic arguments
+
+For consistency with other methods, `order_by` and `group_by` now accept
+variadic arguments:
+
+```{python}
+t.order_by(t["body_mass_g"].desc(), "island", t["species"])
+```
+
+```{python}
+grouped = (
+    t.group_by("species", "island")
+    .agg(count=t.count())
+    .order_by(ibis._["count"].desc())
+)
+grouped
+```
+
+You can still pass in a list or tuple if you prefer.
+
+### Create more backends with empty URLs
+
+For convenience, you can now create backends with an empty URL for Dask, pandas,
+Polars, and DataFusion:
+
+```{python}
+con = ibis.connect("dask://")
+con = ibis.connect("pandas://")
+con = ibis.connect("polars://")
+con = ibis.connect("datafusion://")
+```
+
+This is in addition to the existing backend-specific connection methods:
+
+```{python}
+con = ibis.dask.connect()
+con = ibis.pandas.connect()
+con = ibis.polars.connect()
+con = ibis.datafusion.connect()
+```
+
+Note that URL parameters are not meaningful with these backends. The main use
+case is more convenient connection swapping programmatically:
```

Comment:
What was the issue with `getattr(ibis, backend).connect()`? 🤷 I guess `getattr` feels more "advanced"?

Reply:
yeah nobody knows how to do that